tests/attachment/delete_test.php

```diff
@@ -103,25 +118,24 @@ public function data_attachment_unlink()
 	 */
 	public function test_attachment_delete_success($remove_success, $exists_success, $expected, $throw_exception = false)
 	{
-		$this->filesystem = $this->createMock('\phpbb\filesystem\filesystem', array('remove', 'exists'));
+		$this->storage = $this->createMock('\phpbb\storage\storage', array('delete', 'exists'));
 		if ($throw_exception)
 		{
-			$this->filesystem->expects($this->any())
-				->method('remove')
-				->willThrowException(new \phpbb\filesystem\exception\filesystem_exception);;
+			$this->storage->expects($this->any())
+				->method('delete')
+				->willThrowException(new \phpbb\storage\exception\exception);;
```

Comment:
How could this work?

Reply:
It does. Remove duplicate semi colons pls.